app/services/scrapers/parties/low_priority.rb

```diff
@@ -0,0 +1,31 @@
+module Scrapers
+  module Parties
+    class LowPriority
+      attr :parties_oscn_ids
+
+      def initialize(days_ago: 90, limit: low_count)
+        @parties_oscn_ids = ::Party.older_than(days_ago.days.ago).limit(limit).select(:oscn_id)
+      end
+
+      def self.perform(days_ago: 90)
+        new(days_ago: days_ago).perform
```

Comment:
it looks like we're using the service object pattern with `self.perform` pretty consistently in the code. It think we can move it into its own base service

Reply:
will do in a later PR